modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/services/AccountsService.java

```diff
@@ -50,33 +63,37 @@
     private final WebSocketManager wsm;
 
     /** */
-    private final MailService mailSrvc;
+    private final NotificationService notificationSrvc;
 
     /** */
     private final PasswordEncoder encoder;
 
+    /** User details checker. */
+    private UserDetailsChecker userDetailsChecker;
+
     /** */
-    @Value("${app.activation.enabled}")
+    @Value("${app.activation.enabled:false}")
     private boolean activationEnabled;
 
     /**
      * @param txMgr Transactions manager.
      * @param accountsRepo Accounts repository.
      * @param wsm Websocket manager.
-     * @param mailSrvc Mail service.
+     * @param notificationSrvc Mail service.
      */
     public AccountsService(
         TransactionManager txMgr,
         AccountsRepository accountsRepo,
         WebSocketManager wsm,
-        MailService mailSrvc
+        NotificationService notificationSrvc
     ) {
         this.txMgr = txMgr;
         this.accountsRepo = accountsRepo;
         this.wsm = wsm;
-        this.mailSrvc = mailSrvc;
+        this.notificationSrvc = notificationSrvc;
 
         this.encoder = encoder();
+        this.userDetailsChecker = checker();
```

Comment:
I do not like that we execute non static methods from constructor.
Lets move encoder and checker to some other place and inject here properly?